pkg/reconciler/mutatingwebhook/mutatingwebhook_controller.go

```diff
@@ -0,0 +1,81 @@
+package reconciler
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"k8s.io/api/admissionregistration/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openservicemesh/osm/pkg/certificate"
+	"github.com/openservicemesh/osm/pkg/constants"
+	"github.com/openservicemesh/osm/pkg/logger"
+)
+
+var log = logger.New("mutatingwebhook-reconciler")
```

Comment:
I'd create a logger for the reconciler package to be consistent with existing logging facility, where we use package scoped loggers.

Reply:
I've updated the logger to be package scoped